en-US/GetStarted/Prototype/SetupDevice.htm

```diff
@@ -2,13 +2,13 @@
 layout: getstarted-steps-prototype
 title: Windows 10 IoT Core Official Website | Developer Resource
 description: Windows 10 IoT Core for your Internet of Things. 
-keyword: windows iot, get started, setup device, iot, intel, qualcomm, NOOBS
+keyword: windows iot, get started, home page, iot
```

Comment:
Why remove the keywords ? I think these get used for search engine optimizations.

Reply:
Homepage wasn't a correct keyword for the set up device page.

Reply:
Well in this change you are removing the right keywords and adding home page.